runtimes/pythonrt/pythonrt.go

```diff
@@ -140,12 +140,32 @@ func (py *pySvc) Build(ctx context.Context, fs fs.FS, path string, values []sdkt
 		return sdktypes.InvalidBuildArtifact, err
 	}
 
+	exports, err := pyExports(ctx, py.pyExe, fs)
+	if err != nil {
+		py.log.Error("get exports", zap.Error(err))
+		return sdktypes.InvalidBuildArtifact, err
+	}
+
+	buildExports := kittehs.Transform(exports, func(e Export) sdktypes.BuildExport {
```

Comment:
Don't learn from Itay. Long anonymous functions and 3 levels of indentation are not cool. :)

It's up to you if you want to keep this as-is, but please be thoughtful about this in the future.

Reply:
Moved to external function.